study.md

```diff
@@ -21,7 +21,9 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Write the Ruby code that takes the integer `700` and returns the string `"007"`.
 
 ```ruby
-# your answer here
+let num = "700"
```

Comment:
This will cause a syntax error since `let` isn't a keyword in Ruby.  `700.to_s.reverse`.